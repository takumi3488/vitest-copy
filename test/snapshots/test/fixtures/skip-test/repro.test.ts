import { describe, expect, it } from 'vitest'

const ENABLE_SKIP = process.env.ENABLE_SKIP;

describe.skipIf(ENABLE_SKIP)('repro suite', () => {
  it('inner case', () => {
    expect('hi-1').toMatchSnapshot()
  })
})

it.skipIf(ENABLE_SKIP)('top-level case', () => {
  expect('hi-2').toMatchSnapshot()
})

// at least one non-skipped test is needed to reproduce a bug.
// without this, there will be no SnapshotClient.startCurrentRun,
// so the code to check skip/obsolete snapshot is not exercised.
it('normal case', () => {
  expect(0).toBe(0)
})
