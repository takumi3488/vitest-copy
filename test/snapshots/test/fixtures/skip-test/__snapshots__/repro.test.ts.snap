// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`repro suite > inner case 1`] = `"hi-1"`;

exports[`top-level case 1`] = `"hi-2"`;
